docs/installation.rst

```diff
@@ -1,17 +1,25 @@
 Getting Started
 ===============
 
+Pre-Requisites
+--------------
 
-In order to install the correct modules run this code in the terminal, however if you use a different package manager than pip, change accordingly
+Firstly you must ensure that on the system that you are using, and whether you are using a virtual environment or intend to do everything globally, you must not use a python version higher than 3.10 as these are the versiosns that the game has been been tested to work optimally with and if not some packages will not install such as 'rembg', versions between 3.8 and 3.10 should run with no issues, preferably install version 3.10 in a virtual environment. 
+
+Secondly, from the github repository https://github.com/sussex-Team15/Settler-Project/tree/main, you must download the repository as a zip file to your machine and save in an appropriate location.
```

Comment:
This step is not exactly true because they can either download it in a zip or clone it it doesn't matter